packages/admin/admin/src/mui/menu/Context.tsx

```diff
@@ -3,17 +3,22 @@ import * as React from "react";
 export interface IMenuContext {
     open: boolean;
     toggleOpen: () => void;
+    drawerVariant: "temporary" | "permanent";
+    setDrawerVariant: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
Why do we need this as part of the context?

Reply:
I added this to be able to access the drawerVariant. It is needed in the CollapsibleItem for rendering purposes. See [65cb263](https://github.com/vivid-planet/comet/pull/1240/commits/65cb26310aebd3a9e51ec57194565c079e384193). Without it the context menu rendered unintentionally in some cases.